deployment/environment/devenv/README.md

```diff
@@ -14,11 +14,11 @@ Pre-requisites:
 
 #### Setting Up Testconfig with Simulated Private Ethereum Network
 
-To run tests (e.g., [ccip-test](../../integration-tests/smoke/ccip_test.go)), 
-you need to set up the testconfig following the [testconfig setup instructions](../../integration-tests/testconfig/README.md). 
+To run tests (e.g., [ccip-test](../../../integration-tests/ccip-tests/smoke/ccip_test.go)), 
```

Comment:
The reference to ccip_test.go in this readme should be updated as you have removed it.

Reply:
I believe this was referencing the <= v1.5 smoke test, not the v1.6 one

Reply:
This reference was broken before, I just fixed it here

Reply:
I doubt that as <= v1.5 has its own readme. @AnieeG Tagging you as you have added this file initially.